src/devices/amina.ts

```diff
@@ -0,0 +1,357 @@
+                    offlineCurrent: {ID: aminaControlAttributes.offline_current, type: Zcl.DataType.UINT8},
+                    offlineSinglePhase: {ID: aminaControlAttributes.offline_single_phase, type: Zcl.DataType.UINT8},
+                    timeToOffline: {ID: aminaControlAttributes.time_to_offline, type: Zcl.DataType.UINT16},
+                    enableOffline: {ID: aminaControlAttributes.enable_offline, type: Zcl.DataType.UINT8},
+                    totalActiveEnergy: {ID: aminaControlAttributes.total_active_energy, type: Zcl.DataType.UINT32},
+                    lastSessionEnergy: {ID: aminaControlAttributes.last_session_energy, type: Zcl.DataType.UINT32},
+                },
+                commands: {},
+                commandsResponse: {},
+            }),
+
+            onOff({
+                powerOnBehavior: false,
+            }),
+
+            numeric({
+                name: 'charge_limit',
+                cluster: 'genLevelCtrl',
+                attribute: 'currentLevel',
+                description: 'Maximum allowed amperage draw',
+                reporting: {min: 0, max: 'MAX', change: 1},
+                unit: 'A',
+                valueMin: 6,
+                valueMax: 32,
+                valueStep: 1,
+                access: 'ALL',
+            }),
+
+            numeric({
+                name: 'total_active_power',
+                cluster: 'haElectricalMeasurement',
+                attribute: 'totalActivePower',
+                description: 'Instantaneous measured total active power',
+                reporting: {min: '10_SECONDS', max: 'MAX', change: 10},
+                unit: 'kW',
+                scale: 1000,
+                precision: 2,
+                access: 'STATE_GET',
+            }),
+
+            numeric({
+                name: 'total_active_energy',
+                cluster: 'aminaControlCluster',
+                attribute: 'totalActiveEnergy',
+                description: 'Sum of consumed energy',
+                //reporting: {min: '10_SECONDS', max: 'MAX', change: 5}, // Not Reportable atm
+                unit: 'kWh',
+                scale: 1000,
+                precision: 2,
+                access: 'STATE_GET',
+            }),
+
+            numeric({
+                name: 'last_session_energy',
+                cluster: 'aminaControlCluster',
+                attribute: 'lastSessionEnergy',
+                description: 'Sum of consumed energy last session',
+                //reporting: {min: '10_SECONDS', max: 'MAX', change: 5}, // Not Reportable atm
+                unit: 'kWh',
+                scale: 1000,
+                precision: 2,
+                access: 'STATE_GET',
+            }),
+
+            binary({
+                name: 'ev_connected',
+                cluster: 'aminaControlCluster',
+                attribute: 'evConnected',
+                description: 'An EV is connected to the charger',
+                valueOn: ['True', 1],
```

Comment:
```suggestion
                valueOn: ['true', 1],
```

Please make these lower case (also for the others)

Reply:
Fixed in https://github.com/Koenkk/zigbee-herdsman-converters/pull/8191/commits/c96eadaf875ad54223477f3a1774c8013ede8384